firefox-ios/Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -223,17 +224,48 @@ final class LaunchCoordinator: BaseCoordinator,
         router.present(viewController, animated: false)
     }
 
+    private lazy var onboardingService: OnboardingService = {
+        OnboardingService(
+            windowUUID: windowUUID,
+            profile: profile,
+            themeManager: themeManager,
+            delegate: self,
+            navigationDelegate: self,
+            qrCodeNavigationHandler: self
+        )
+    }()
+
     // MARK: - Intro
     private func presentModernIntroOnboarding(with manager: IntroScreenManager,
                                               isFullScreen: Bool) {
         let onboardingModel = NimbusOnboardingKitFeatureLayer().getOnboardingModel(for: .freshInstall)
+        let activityEventHelper = ActivityEventHelper()
+        let telemetryUtility = OnboardingTelemetryUtility(with: onboardingModel)
+        let introScreenManager = IntroScreenManager(prefs: profile.prefs)
```

Comment:
Why are we not using `manager` here and create a new `IntroScreenManager` instead of using the one that is passed in?